filechanges/etc/samba/smb.conf

```diff
@@ -86,7 +86,7 @@ follow symlinks = yes
     directory mask = 0777
 
 #[Webclient]
-#    path = /opt/defaultwebclient
+#    path = /opt/webclient
```

Comment:
My guess is it's left over from when you needed to editing js/functionsvars.js to edit the radiostations. In which case we can get rid of this (in another PR).